### DISCUSSION TITLE
'@infragistics/igniteui-angular@17.1.11' is not in this registry.

### DISCUSSION THREAD

**Discussion by Fisher1993** [IgniteUI/igniteui-angular#14211] [Q&A] [answered]
Hi,

I am getting this error while running 'npm install':
'@infragistics/igniteui-angular@17.1.11' is not in this registry.

 Please help.

**Comment by MonikaKirkova** [comment, marked as answer]
Hi @Fisher1993,

This error message indicates that the package is not found in the registry, and I assume that it is searched in the default npm registry. However, the licensed version of the product should be accessed from our private feed. A step by step guide on how to setup your environment to use the private npm feed could be found [in the following topic of our documentation](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/ignite-ui-licensing#how-to-setup-your-environment-to-use-the-private-npm-feed-step-by-step-guide).
